Command/FindGithubRepositoryThroughOrganizationCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace OpenCatalogi\OpenCatalogiBundle\Command;
+
+use OpenCatalogi\OpenCatalogiBundle\Service\FindGithubRepositoryThroughOrganizationService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Command to execute the FindGithubRepositoryThroughOrganizationService
+ */
+class FindGithubRepositoryThroughOrganizationCommand extends Command
+{
+    protected static $defaultName = 'opencatalogi:findGithubRepositoryThroughOrganization:execute';
```

Comment:
Command defaultName is not in line with other opencatalogi commands, i would start with removing :execute at the end. compare naming with: https://github.com/OpenCatalogi/OpenCatalogiBundle/pull/10/files